### ISSUE TITLE
Usage unclear for users new to Somfy IO

### ISSUE THREAD

**Issue by shapecoder** [tetienne/somfy-open-api#38]
Hi, I'm going to buy several Somfy IO roller shutters and would like to control them using this library. I'm currently at the shop that sells them trying to interface with their TaHoma box, but I just can't get it to work.

I followed the readme and created a developer account with my own email address and an app which has been authorized for access. When I try to retrieve devices it just returns an empty list, so far so good.

Now I think I need to connect my app to the TaHoma box from the shop, but I just can't figure out how to do it. I looked around in the TaHoma HD app, but I can't find a way to add my app, or create a key or something. I can open the Somfy account from the app, but that only displays the box and when I open it I can only buy SMS alerts.

The FAQ (https://developer.somfy.com/faq-page) has an answer for: "The APIs give access to Somfy products control. What is the process to be followed by users before this control phase to be effective in your app?": First, the users must use the Somfy applications to create their accounts, activate the products and pair end devices with the Somfy hubs (like Somfy motorizations to be paired with TaHoma or Connexoon). Once this is completed and the integration of Somfy APIs in your app done, users can connect their Somfy accounts to your app (through the account linking to be found in your app).

But I can't find the account linking... How do I connect my app to the TaHoma box? Do I need to create a developer account with the same email address which was used for the Somfy account to which the box is registered or something? This could be more clear in the readme.

**Comment by tetienne** [comment]
Hi,

Oauth2 process can be confusing at first look indeed. 

Once you have created your app  within Somfy developer website, update the variable `client_id`, `redir_url` and `secret` from the README example. Don't forget also to set a valid path in `cache_path`. 

Launch the script. You will see a message in the console asking you to go on a specific URL. Do it. You will see the Somfy login portal. On this one, you will have to log in using the same credentials you use to log in on your Tahoma. 

Once log in, you will be redirected to the URL you set in your app. Don't worry if you got a 404, just copy the new URL in your browser. This one contains the token. Paste it in the console. 

You will see your covers displayed.

**Comment by shapecoder** [comment]
Yeah, I managed to figure it out. What you described was clear to me, but it wasn't clear to me that the developer account should be the same account as the TaHoma account (I was trying to access the TaHoma registered to the account of the shop with my personal developer account, that didn't work). Once I got the credentials from the shop it took only a few minutes to figure it out :)

Wait, actually, you're saying that the two accounts don't have to be the same as long as you use the account registered with the TaHoma box when authorizing the URL that the script spits out? Well, at least I think I understand it now :)
